testsuite/features/step_definitions/command_steps.rb

```diff
@@ -538,8 +545,10 @@
     cmd = "zypper mr --disable #{repo}"
   elsif file_exists?(node, '/usr/bin/yum')
     cmd = "sed -i 's/enabled=.*/enabled=0/g' /etc/yum.repos.d/#{repo}.repo"
+  elsif file_exists?(node, '/usr/bin/apt-get')
+    cmd = "sed -i '/^deb.*/ s/^deb /#deb /' /etc/apt/sources.list.d/#{repo}.list"
   else
-    raise 'Not found: zypper or yum'
+    raise 'Not found: zypper or yum or apt-get'
```

Comment:
Correct English would be: `Not found: zypper, yum, or apt-get`

Same below (3 occurrences on the whole)